### PR TITLE
fix docs: add the usage of kubectl vela show in the docs

### PR DESCRIPTION
**Pull request by yangsoon**
as title

### CODE REVIEW

File: charts/vela-core/templates/defwithtemplate/cpuhpa.yaml [hunk 1]
```diff
@@ -38,8 +38,14 @@ spec:
...
-        	min:     *1 | int
-        	max:     *10 | int
+        
+        	// +usage=specify the min number of replicas to which the autoscaler can scale down
+        	min: *1 | int
+        
+        	// +usage=specify the max number of of replicas to which the autoscaler can scale up
+        	max: *10 | int
+        
+        	// +usage=specify the average cpu utilization
```
Comment (zzxwill) [inline charts/vela-core/templates/defwithtemplate/cpuhpa.yaml:48]:
```suggestion
        	// +usage=specify the average cpu utilization, for example, 50 means 50%
```

Developers often get confused by the meaning of usage, so an example can help:)

File: charts/vela-core/templates/defwithtemplate/cpuhpa.yaml [hunk 2]
```diff
@@ -38,8 +38,14 @@ spec:
         	}
         }
         parameter: {
-        	min:     *1 | int
-        	max:     *10 | int
+        
+        	// +usage=specify the min number of replicas to which the autoscaler can scale down
```
Comment (zzxwill) [inline charts/vela-core/templates/defwithtemplate/cpuhpa.yaml:42]: Suggest not to use `min` as you are explaining `min` at the very moment.

Comment (yangsoon) [reply charts/vela-core/templates/defwithtemplate/cpuhpa.yaml:42]: thanks! I will fix it.

File: charts/vela-core/templates/defwithtemplate/cpuhpa.yaml (hunk 1 above)
Comment (yangsoon) [reply charts/vela-core/templates/defwithtemplate/cpuhpa.yaml:48]: got it

File: charts/vela-core/templates/defwithtemplate/manualscale.yaml [hunk 3]
```diff
@@ -18,7 +18,8 @@ spec:
         	spec: replicas: parameter.replicas
         }
         parameter: {
-        	//+short=r
+        	// +usage=specify the number of workload
+        	// +short=r
```
Comment (zzxwill) [inline charts/vela-core/templates/defwithtemplate/manualscale.yaml:22]: `+short` can be deleted as cli trait operation is not supported any more.

File: charts/vela-core/templates/defwithtemplate/cpuhpa.yaml (hunk 2 above)
Comment (zzxwill) [inline charts/vela-core/templates/defwithtemplate/cpuhpa.yaml:42]: Look the generated docs or cli table, capitalizing the first word look better:)

**Review by zzxwill — APPROVED** [review]
This PR will help a lot for application developers:) Great job.

File: charts/vela-core/templates/defwithtemplate/manualscale.yaml (hunk 3 above)
Comment (yangsoon) [reply charts/vela-core/templates/defwithtemplate/manualscale.yaml:22]: got it

File: charts/vela-core/templates/defwithtemplate/cpuhpa.yaml (hunk 2 above)
Comment (yangsoon) [reply charts/vela-core/templates/defwithtemplate/cpuhpa.yaml]: got it, align with other definition is necessary

**Review by wonderflow — APPROVED** [review]
good job

**Pull request merged**